### PR TITLE
[CELEBORN-427] Add doc for http request REST API

### PR DESCRIPTION
**Pull request by AngersZhuuuu**
127 0 0 1_8000_user_guide_monitoring_

### CODE REVIEW

**Comment by AngersZhuuuu** [comment]
ping @pan3793 @waitinfuture @FMX @nafiyAix

File: docs/user_guide/monitoring.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+# Metrics
+
+# REST API
+
+In addition to viewing the metrics, Celeborn also support REST API. This gives developers
+an easy way to create new visualizations and monitoring tools for Celeborn and
+also easy for users to get the running status of the service. The REST API is available for
+both master and worker. The endpoints are mounted at `host:port`. For example,
+for the master, they would typically be accessible
+at `http://<master-prometheus-host>:<master-prometheus-port><endpoint>`, and
```
Comment (waitinfuture) [inline docs/user_guide/monitoring.md]: slash before <endpoint>

File: docs/user_guide/monitoring.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+for the worker, at `http://<worker-prometheus-host>:<worker-prometheus-port><endpoint>`.
```
Comment (waitinfuture) [inline docs/user_guide/monitoring.md]: ditto

File: docs/user_guide/monitoring.md (hunk 1 above)
Comment (waitinfuture) [inline docs/user_guide/monitoring.md]: IMO we'd better use "path" instead of "endpoint", which is URI's convention

Comment (AngersZhuuuu) [reply docs/user_guide/monitoring.md]: Done

File: docs/user_guide/monitoring.md (hunk 2 above)
Comment (AngersZhuuuu) [reply docs/user_guide/monitoring.md]: Done

File: docs/user_guide/monitoring.md (hunk 1 above)
Comment (AngersZhuuuu) [reply docs/user_guide/monitoring.md]: DOne

**Review by waitinfuture — APPROVED** [review]
LGTM, thanks!

**Pull request merged**